packages/block-library/src/post-featured-image/style.scss

```diff
@@ -2,4 +2,8 @@
 	a {
 		display: inline-block;
 	}
+	img {
+		max-width: 100%;
+		height: auto;
+	}
```

Comment:
This makes sense, no? Is there a way to handle the images?

Reply:
That makes sense. 👍